### ISSUE TITLE
[Feature] Automatically get rid of containing folders

### ISSUE THREAD

**Issue by dfourn** [lardbit/nefarious#52]
After it downloads a season of episodes, I would rather the season folder contain just the video files and subtitles rather than a collection of folders for each episode.

**Comment by lardbit** [comment]
Yeah, I agree.  I haven't really fleshed out the best way to do that yet.  I'm not sure how to handle torrents with an unknown amount of sub-directories and duplicate file names in those nested directories.  The normal cases are simple enough, but I'm not sure about the edge cases.

I'll keep this open until I get the chance to tackle this.  If you have any ideas or examples of other people doing it, send it along.

**Comment by dfourn** [comment]
Sorry about a slow reply. I'm pretty sure sonarr does it, although probably not in a default setting. I would implement it in a safe way. Have a few possible scenarios that fit the most common possibilities and only if it matches one of these scenarios, we can move the video and subtitle files out and delete folder and an info text file.
